projectile.el

```diff
@@ -1203,8 +1203,13 @@ Return the first (topmost) matched directory or nil if not found."
   "Identify a project root in DIR by bottom-up search for files in LIST.
 If LIST is nil, use `projectile-project-root-files-bottom-up' instead.
 Return the first (bottommost) matched directory or nil if not found."
-  (cl-some (lambda (name) (projectile-locate-dominating-file dir name))
-           (or list projectile-project-root-files-bottom-up)))
+  (projectile-locate-dominating-file
+   dir
+   (lambda (directory)
+     (when (file-readable-p directory)
```

Comment:
Do we really need those permissions checks? I get their purpose, but we generally assume that most of the time you'd be dealing with something readable.

Reply:
I’m happy to get rid of this one. The one below here should probably be `file-exists-p` anyway. We don’t really care if, say, `.git` is readable, if it exists, it marks a project.